source/fundamentals/indexes.txt

```diff
@@ -361,7 +361,7 @@ The following is an example of a geospatial query using the "location.geo" index
 
 MongoDB also supports ``2d`` indexes for calculating distances on a Euclidean plane and for working with the "legacy
 coordinate pairs" syntax used in MongoDB 2.2 and earlier. See the :manual:`Geospatial Queries page </geospatial-queries>`
-in the MongoDB server manual for more information.
+in the MongoDB Server manual for more information.
```

Comment:
[suggestion]

Per our [Style Guide](https://www.mongodb.com/docs/meta/style-guide/style/links-and-cross-references/#make-link-text-clear), we should put the reason why we have the link before we say "see X".